### PR TITLE
change to avy

### PR DESCRIPTION
**Pull request by samontea**
solves #101

### CODE REVIEW

File: .emacs.d/lisp/behavior/text.el [hunk 1]
```diff
@@ -7,12 +7,12 @@
 (require 'cl)
 
 ;;; ace jump mode
```
Comment (rye) [inline .emacs.d/lisp/behavior/text.el]: Comment is now stale

File: .emacs.d/lisp/package/dependencies.el [hunk 2]
```diff
@@ -11,8 +11,8 @@
     auto-complete ;; code autocompletion
     markdown-mode ;; major mode for markdown
     smart-tabs-mode ;; indentation with tabs and spaces
-    ace-jump-mode ;; immediately jump to any visible character
-    ace-jump-zap ;; zap-to-char but harnessing ace-jump
+    avy ;; immediately jump to any visible character
+    avy-zap ;; zap-to-char but harnessing ace-jump
```
Comment (rye) [inline .emacs.d/lisp/package/dependencies.el]: Comment is stale

**Review by rye — CHANGES_REQUESTED** [review]
LGTM with nits

**Review by cg505 — APPROVED** [review]
lgtm (except for @rye's nits)

**Pull request merged**